common/postgresql/templates/deployment.yaml

```diff
@@ -53,8 +53,39 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /postgresql
+{{- if .Values.backup.enabled }}
+      - image: {{.Values.backup.repository}}:{{.Values.backup.image_version}}
+        name: backup
+        env:
+          - name: MY_POD_NAME
+            value: postgres-backup-{{.Values.name}}
+          - name: MY_POD_NAMESPACE
+            value: {{.Release.Namespace}}
+          - name: OS_AUTH_URL
+            value: {{.Values.backup.os_auth_url}}
+          - name: OS_AUTH_VERSION
+            value: {{.Values.backup.os_auth_version | quote}}
+          - name: OS_IDENTITY_API_VERSION
```

Comment:
What is the difference between `OS_AUTH_VERSION` and `OS_IDENTITY_API_VERSION`? Do we really need both?